ui/ducks/bridge/bridge.ts

```diff
@@ -36,6 +36,15 @@ const bridgeSlice = createSlice({
     setFromTokenInputValue: (state, action) => {
       state.fromTokenInputValue = action.payload;
     },
+    resetInputFields: () => ({
+      ...initialState,
+    }),
+    switchToAndFromTokens: (state, { payload }) => ({
+      toChainId: payload,
+      fromToken: state.toToken,
+      toToken: state.fromToken,
+      fromTokenInputValue: null,
```

Comment:
We do this in the Portfolio Bridge experience to avoid showing potential errors (example: insufficient balance) right when the user switches their tokens. This also follows the pattern used by Send, where the input amount is cleared when the user changes the src token

I'm open to either approach tho and seems like the aggregators don't have a consistent pattern - maybe we can loop in Varada?

Also fyi switching tokens only partially works as of this PR. Since it involves updating the  quote request, the working version will be in the quote-fetching PR

Reply:
Yea let's loop in Design. I don't necessarily think this is a blocker for this PR though. I'm fine with addressing it in another.

Reply:
Sounds good, will ask Varada about this separately